code/modules/medical/genetics/bioEffects/beneficial.dm

```diff
@@ -255,6 +255,31 @@
 	heal_per_tick = 7 // decrease to 5 if extreme narcolepsy doesn't counterbalance this enough
 	regrow_prob = 50 //increase to 100 if not counterbalanced
 
+/datum/bioEffect/regenerator/wolf
+	name = "Lupine Regeneration"
+	desc = "Subject's cells are programmed to reshape itself into a canine form."
+	id = "regenerator_wolf"
+	occur_in_genepools = 0
+	probability = 0
+	scanner_visibility = 0
+	can_research = 0
+	can_make_injector = 0
+	can_copy = 0
+	can_reclaim = 0
+	can_scramble = 0
+	curable_by_mutadone = 0
+	stability_loss = 0
+	msgGain = "You feel oddly feral."
+	msgLose = "You feel more comfortable in your own skin."
+	heal_per_tick = 2
+	regrow_prob = 50
+
+	OnLife()
```

Comment:
Good point, lycanthropy already checks if someone's a wolf, and leaves them be if they are. And the proc that makes the player an antag wolf also gives them resistance to the disease. Since the only way this wouldnt add the disease is if they were admin transformed into a wolf, then given the bioeffect... yeah that'd work a lot better.